app/helpers/application_helper.rb

```diff
@@ -10,29 +10,143 @@ def login_helper(style = '')
+      },
+      {
+        url: 'path-to-be-added',
+        title: 'Team hub'
+      },
+      {
+        url: 'path-to-be-added',
+        title: 'Lifestyle'
+      },
+      {
+        url: 'path-to-be-added',
+        title: 'Dealbook'
+      },
+      {
+        url: 'path-to-be-added',
+        title: 'Video'
+      }
+    ]
+  end
+
+  def admin_side_panel_items
+    src_path = 'admin-side-panel/'
+    [
+      {
+        url: 'path-to-be-added',
+        title: 'Surveys',
+        svg_path: src_path + 'surveys.svg'
+      },
+      {
+        url: 'path-to-be-added',
+        title: 'Banners',
+        svg_path: src_path + 'banners.svg'
+      },
+      {
+        url: 'path-to-be-added',
+        title: 'Languages',
+        svg_path: src_path + 'languages.svg'
+      },
+      {
+        url: 'path-to-be-added',
+        title: 'Footer',
+        svg_path: src_path + 'footer.svg'
+      },
+      {
+        url: 'path-to-be-added',
+        title: 'Social Networks',
+        svg_path: src_path + 'social-networks.svg'
+      },
+      {
+        url: 'path-to-be-added',
+        title: 'Users',
+        svg_path: src_path + 'users.svg'
+      },
+      {
+        url: 'path-to-be-added',
+        title: 'IA',
+        svg_path: src_path + 'ia.svg'
+      },
+      {
+        url: 'path-to-be-added',
+        title: 'Teams',
+        svg_path: src_path + 'teams.svg'
+      },
+      {
+        url: 'path-to-be-added',
+        title: 'News Partners',
+        svg_path: src_path + 'news-partners.svg'
+      },
+      {
+        url: 'path-to-be-added',
+        title: 'Advertising',
+        svg_path: src_path + 'advertising.svg'
       }
     ]
   end
 
-  def nav_helper(style, tag_type)
-    nav_links = ''
+  # Todo: replace this mock with actual Sports when they are added
+  def mock_sports
+    [
+      {
+        url: 'path-to-be-added',
+        title: 'NBA'
+      },
+      {
+        url: 'path-to-be-added',
+        title: 'NFL'
+      },
+      {
+        url: 'path-to-be-added',
+        title: 'NASCAR'
+      }
+    ]
+  end
 
-    nav_items.each do |item|
-      nav_links << "<#{tag_type}><a href='#{item[:url]}' class='#{style} #{active? item[:url]}'>#{item[:title]}</a></#{tag_type}>"
+  def side_panel_helper (style, tag_type, is_admin_page: false)
+    panel_links = ''
+
+    panel_items = side_panel_items.insert(1, *mock_sports)
+    panel_items.each do |item|
+      link_url = is_admin_page ? request.path + item[:url] : item[:url]
+      panel_links << "<#{tag_type}><a href='#{link_url}' class='#{style} #{active? link_url}'>#{item[:title]}</a></#{tag_type}>"
     end
 
-    nav_links.html_safe
+    panel_links.html_safe
+  end
+
+  def admin_side_panel_helper (style, tag_type)
+      panel_links = ''
+
+      admin_side_panel_items.each do |item|
+        panel_links << "<#{tag_type}><a href='#{item[:url]}' class='#{style} #{active? item[:url]}' title='#{item[:title]}' data-bs-toggle='tooltip' data-bs-placement='right'>
```

Comment:
`active?` need to return `true` or `false` otherwise there should be no `?` in method name

also check tag helpers https://api.rubyonrails.org/v6.1.4/classes/ActionView/Helpers/TagHelper.html